e2e/pages/searchCHPL/listingEdit/listingEdit.po.js

```diff
@@ -0,0 +1,106 @@
+class ListingEditPage {
+    constructor () { }
+
+    get editcertifiedProduct () {
+        return $('#inspect-edit');
+    }
+
+    get testProcedureName () {
+        return $('//*[@id="testProcedures "]');
+    }
+
+    get allTestProcedureVersion () {
+        return $$('*[id^="testProcedures-additional-input"]');
+    }
+
+    get testToolsName () {
+        return $('//*[@id="testTools "]');
+    }
+
+    get alltestToolsVersion () {
+        return $$('*[id^="testTools-additional-input"]');
+    }
+
+    get testDataName () {
+        return $('//*[@id="testData "]');
+    }
+
+    get alltestDataVersion () {
+        return $$('*[id^="testData-additional-input"]');
+    }
+
+    get testFunctionalityName () {
+        return $('*[id="testFunctionality "]');
+    }
+
+    get saveCertifiedProduct () {
+        return $('button=Save Certification Criteria');
+    }
+
+    get closeListingEditBtn () {
+        return $('button.close.pull-right.ng-isolate-scope');
+    }
+
+    get yesConfirmation () {
+        return $('//button[text()="Yes"]');
+    }
+
+    gotoEditCriteria (editCriteriaId , cures) {
+        this.editcertifiedProduct.click();
+        if (cures === true) {
```

Comment:
`cures` is a boolean, right? Why not
```suggestion
        if (cures) {
```